bee-protocol/src/worker/transaction/transaction.rs

```diff
@@ -129,51 +130,59 @@ impl TransactionWorker {
         }
 
         // store transaction
-        match tangle().insert_transaction(transaction, hash).await {
-            Some(transaction) => {
-                Protocol::get().metrics.new_transactions_received_inc();
```

Comment:
Did this just disappear ?

Reply:
Oopsie! Did I accidentally press `dj`? That would have probably never happened if I just used Emacs :trollface: 
Anyways. Re-added! Thanks.